src/components/ChatRoom/ChatRoom.js

```diff
@@ -12,11 +12,13 @@ class ChatRoom extends Component {
 	constructor(props) {
 		super(props);
 		this.state = { message: '' };
-		this.socket = io('http://thawing-chamber-88612.herokuapp.com/');
```

Comment:
We should create an import folder for our servers so it's easily changeable/readable.